docs/using.rst

```diff
@@ -152,9 +108,9 @@ To use ADB with Host-Based tools:
    docker CLI to access the docker daemon.
 
    **Note:** When the OpenShift service is running in the VM, a docker registry
-   is also started. This Docker registry can be consumed by external
-   tools such as Eclipse to push or pull images. The Docker registry url is exported
-   as a variable, and can be accessed as shown below::
+   is also started. This Docker registry can be used by external tools such
+   as Eclipse to push or pull images. The Docker registry url is exported as a
+   variable, and can be accessed as shown below::
```

Comment:
we try to avoide above/below (these are terms left over from the PDF/book days :), consider "by running the following command"